src/test/java/ca/on/oicr/gsi/dimsum/MockCase.java

```diff
@@ -438,9 +469,35 @@ private static Case makeCase(String donorName, String assayName, String projectN
     addRequisition(kase, caseNumber, requisitionName);
     when(kase.getDeliverables()).thenReturn(new ArrayList<>());
     addDeliverable(kase, DeliverableType.DATA_RELEASE, "Full Pipeline");
+    when(kase.getStartDate()).thenReturn(startDate);
+    addCompletionDate(kase, completionDate);
     return kase;
   }
 
+  private static LocalDate addCompletionDate(Case kase, LocalDate completionDate) {
```

Comment:
As is, this method doesn't always do what its name implies, and a lot of release QCs are left without completion dates. Instead, set the date when marking release QC (add date parameter to `markRelease`)